plugins/micromega/csdpcert.mli

```diff
@@ -0,0 +1,9 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
```

Comment:
What does it mean when a .ml exports nothing at all? Doesn't it mean that the .ml file could be removed completely?

Reply:
It's only there for the side effects.

Reply:
Or rather this file is compiled into a standalone executable.

Reply:
OK thanks.